assets/css/edd-admin.css

```diff
@@ -2431,6 +2414,56 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+.edd-admin--has-grid {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	grid-gap: 20px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+
+.edd-admin--has-grid .edd-from-to-wrapper {
+	display: flex;
+	margin-right: 0;
+	margin-bottom: 16px;
+	width: 100%;
+}
+
+.edd-admin--has-grid .edd-from-to-wrapper input {
+	width: 100%;
+}
+
+.edd-admin--has-grid .edd-from-to-wrapper span {
+    flex-grow: 1;
+}
+
+.edd-admin--has-grid .edd-import-export-form {
```

Comment:
With the new class, could this be changed to just `.edd-admin--has-grid form` so it will apply to any form in a grid? That may be too generic, but I'm almost thinking not.